apps/renterd/components/Files/Columns/FilesHealthColumn/FilesHealthColumnContents.tsx

```diff
@@ -91,7 +92,7 @@ export function FilesHealthColumnContents({
             className="flex items-center"
             font="mono"
           >
-            Slab {slab.key.replace('key:', '').slice(0, 4)}:
+            Slab {stripPrefix(slab.encryptionKey).slice(0, 4)}:
```

Comment:
Not terribly important but this could be extracted into some kind of `shorthand` util, seems like we do this for various types and we use varying number of characters. It's consistent though.

Reply:
We have a `truncate` util 👍